mwdb/web/src/components/UserProfile.js

```diff
@@ -80,51 +98,69 @@ class UserProfile extends Component {
                             <td>Last login</td>
                             <td><DateString date={this.state.profile.logged_on}/></td>
                         </tr>
+                        {this.props.userLogin === this.state.profile.login &&
                         <tr>
                             <td>Last password set</td>
                             <td><DateString date={this.state.profile.set_password_on}/></td>
                         </tr>
+                        }
+                        {(this.props.userLogin === this.state.profile.login && this.capabilities.length > 0) &&
```

Comment:
That should be enough
```suggestion
                        {this.capabilities.length > 0 &&
```